Cargo.toml

```diff
@@ -29,6 +29,11 @@ name = "cargo-clippy"
 test = false
 path = "src/main.rs"
 
+[[bin]]
+name = "clippy-driver"
```

Comment:
yesssss this is much better kinda feel like we should have done this on the get go